docker/pythonpath/superset_config.py

```diff
@@ -163,13 +171,38 @@ def oauth_authorized(self, provider: str) -> WerkzeugResponse:
         
         messages = get_flashed_messages(with_categories=True)
         if ('error', translate("The request to sign in was denied.")) in messages:
-            flash(translate("You are not yet authorized to access this application. Please contact a GuardianConnector administrator for access."), "warning")     
+            flash(translate("You are not yet authorized to access this application. Please contact an administrator for access."), "warning")     
         return response
 
+USER_ROLE= get_env_variable("USER_ROLE", "Gamma")
+USER_ROLE_PERMISSIONS = get_env_variable("USER_ROLE_PERMISSIONS", "")
```

Comment:
I can't say I understand what this USER_ROLE(*) stuff does.

Is it changing the Gamma role across the board?

This stuff feels weird to do at deployment time, but ...maybe?

Reply:
What if alice signs up on Monday, when `USER_ROLE_PERMISSIONS = (can_read,Chart)`

Bob signs up Tuesday, when `USER_ROLE_PERMISSIONS = (can_read,Chart),(all_datasource_access),(all_database_access)`

On Wednesday what roles do Alice have?  What roles does Gamma have?

Reply:
I agree about this being weird at deployment time, but it seems to be the established solution. I linked to another Github issue where others have done similar setting of user roles and permissions in `config.py`.

Reply:
> On Wednesday what roles do Alice have? What roles does Gamma have?

Good question. I started to write out a comment to answer the questions, but encountered a few unknowns about what is overwritten vs. persisted in the Superset metastore database, so I will first do some testing.

Reply:
It's probably ok if you don't have answers here.  By acknowledging this you have confirmed that I'm understanding the intent well enough.

Honestly I bet a look in the apache code could be a great way to answer this

Reply:
So the answer is that on Wednesday, both Alice and Bob will have the roles that were set on Tuesday. User role permissions are not assigned to individual users; they are determined by their assigned user role at runtime.

It turns out that any manual changes to user role permissions made in the UI will be overwritten on deployment, per the approach in this PR. But only for the USER_ROLE specified in env: if changes are made to ALPHA and GAMMA in the Superset UI, and the app is redeployed with `USER_ROLE_PERMISSIONS` set for `USER_ROLE` GAMMA, then only GAMMA is reset; the changes to ALPHA will persist.

However, I think this is not a problem for the default user role, and I will document this behavior in the readme.